backend/app/Http/Controllers/Web/Logistics/PalletMovementController.php

```
@@ -0,0 +1,68 @@
+            ->with('success', __('Pallet movement recorded.'));
+    }
+
+    /** Admin movement history — rows arrive client-side via the Electric shape. */
+    public function index()
+    {
+        return Inertia::render('admin/pallet-movements/Index', [
+            // Lookup maps for the live table (rows stream via the shape). Bound
+            // to the ids the ledger actually references so the payload can't
+            // balloon to the whole pallets/workers tables. withTrashed so
+            // soft-deleted pallets/operators still resolve to a label.
+            'operatorNames' => Worker::withTrashed()
```

PHPStan (2.2.5)

[error] Call to an undefined static method App\Models\Worker::withTrashed().


(staticMethod.notFound)